src/libraries/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/Validator.cs

```diff
@@ -303,12 +303,12 @@ public static void ValidateObject(object instance, ValidationContext validationC
         ///         first.
         ///     </para>
         /// </remarks>
-        /// <param name="value">The value to test.  It cannot be null.</param>
+        /// <param name="value">The value to test.</param>
         /// <param name="validationContext">Describes the object being tested.</param>
         /// <param name="validationAttributes">The list of <see cref="ValidationAttribute" />s to validate against this instance.</param>
         /// <exception cref="ArgumentNullException">When <paramref name="validationContext" /> is null.</exception>
         /// <exception cref="ValidationException">When <paramref name="value" /> is found to be invalid.</exception>
-        public static void ValidateValue(object value, ValidationContext validationContext,
+        public static void ValidateValue(object? value, ValidationContext validationContext,
```

Comment:
This doesn't need a corresponding ref change?
https://github.com/dotnet/runtime/pull/91293/files#diff-d3dbf2eac7f872537abf660628dcb3285214f67162c38290fbb428c4492842baR397

Reply:
I think it does need the ref change.

Reply:
@jeffhandley FYI https://github.com/dotnet/runtime/blob/release/8.0/docs/coding-guidelines/updating-ref-source.md